### PR TITLE
Exclude dirs

### PR DESCRIPTION
**Pull request by ae0000**
This will allow you to specify directories you want excluded. I tried https://github.com/phaikawl/fresh which also attempts to add this feature, however it never worked quite right for me.

You add directories you want excluded via the runner.conf :

```
exclude_dir: assets, bower_components, something_else
```

### CODE REVIEW

File: runner/watcher.go [hunk 1]
```diff
@@ -37,12 +38,24 @@ func watchFolder(path string) {
...
+	ex := excludeDir()
+
 	filepath.Walk(root, func(path string, info os.FileInfo, err error) error {
 		if info.IsDir() && !isTmpDir(path) {
 			if len(path) > 1 && strings.HasPrefix(filepath.Base(path), ".") {
 				return filepath.SkipDir
 			}
 
+			if len(ex) > 0 {
+				for _, ep := range strings.Split(ex, ",") {
```
Comment (c2h5oh) [inline runner/watcher.go]: Comma is a valid character in path on some operating systems (Windows for example)

**Comment by metal3d** [comment]
:+1: Really need this option, bower breaks the usage of fresh

**Comment by pilu** [comment]
if @ae0000 fixes the import path we can merge it

**Comment by drKnoxy** [comment]
waiting on this seriously awesome pr

**Comment by gmccue** [comment]
This is fixed in pr #60.

**Pull request closed without merging**